pkg/asset-manager-utils/contracts/RebalancingRelayer.sol

```diff
@@ -76,11 +81,27 @@ contract RebalancingRelayer is IBasePoolRelayer, AssetHelpers {
     function exitPool(
         bytes32 poolId,
         address payable recipient,
-        IVault.ExitPoolRequest memory request
-    ) external rebalance(poolId, request.assets) {
+        IVault.ExitPoolRequest memory request,
+        uint256[] memory minCashBalances
+    ) external rebalance(poolId, request.assets, minCashBalances) {
         vault.exitPool(poolId, msg.sender, recipient, request);
     }
 
+    function _ensureCashBalance(
+        bytes32 poolId,
+        IERC20[] memory tokens,
+        uint256[] memory minCashBalances
+    ) internal {
+        for (uint256 i = 0; i < tokens.length; i++) {
+            (uint256 cash, , , address assetManager) = vault.getPoolTokenInfo(poolId, tokens[i]);
+            uint256 cashNeeded = minCashBalances[i];
```

Comment:
```suggestion
            uint256 cashNeeded = minCashBalances[i];
            if (cashNeeded == 0) continue;
            
            (uint256 cash, , , address assetManager) = vault.getPoolTokenInfo(poolId, tokens[i]);
```

`joinPool` currently passes 0 values, but I imagine we may also have e.g. a single asset exit where we don't really care about the balances of other tokens.

Reply:
This will just be undone when we address #695 (I've got this on a local branch).